Sdk/Talk/SwrveMessageController.m

```diff
@@ -133,12 +133,12 @@ @implementation SwrveMessageController
 
 + (void)initialize {
     ALL_SUPPORTED_DYNAMIC_DEVICE_FILTERS = [NSArray arrayWithObjects:
-        [swrve_permission_location_always stringByAppendingString:swrve_permission_requestable],
-        [swrve_permission_location_when_in_use stringByAppendingString:swrve_permission_requestable],
-        [swrve_permission_photos stringByAppendingString:swrve_permission_requestable],
-        [swrve_permission_camera stringByAppendingString:swrve_permission_requestable],
-        [swrve_permission_contacts stringByAppendingString:swrve_permission_requestable],
-        [swrve_permission_push_notifications stringByAppendingString:swrve_permission_requestable], nil];
+        [[swrve_permission_location_always stringByAppendingString:swrve_permission_requestable] lowercaseString],
+        [[swrve_permission_location_when_in_use stringByAppendingString:swrve_permission_requestable] lowercaseString],
+        [[swrve_permission_photos stringByAppendingString:swrve_permission_requestable] lowercaseString],
+        [[swrve_permission_camera stringByAppendingString:swrve_permission_requestable] lowercaseString],
+        [[swrve_permission_contacts stringByAppendingString:swrve_permission_requestable] lowercaseString],
+        [[swrve_permission_push_notifications stringByAppendingString:swrve_permission_requestable] lowercaseString], nil];
```

Comment:
For clarity, I think it would be better make these they're own independent strings instead of appending requestable to the event name and lowercasing. 

So something like this:
    ALL_SUPPORTED_DYNAMIC_DEVICE_FILTERS = [NSArray arrayWithObjects:
        @"swrve.permission.ios.location.always.requestable",
        etc,
        etc,

They are independent of events, aren't they?